graphql/queryResolvers/moduleCrud.ts

```diff
@@ -0,0 +1,29 @@
+import type { MutationDeleteModuleArgs, MutationAddModuleArgs } from '..'
+import prisma from '../../prisma'
+
+export const modules = async () => {
```

Comment:
You should use typescript for all functions you declare so that you can define what they return